src/apps/relay/windows/ServiceInstaller.cpp

```diff
@@ -0,0 +1,162 @@
+/*!
+ * Install service
```

Comment:
Can you include the code license that the file is distributed under in the comment block at the beginning of each file?